instagram-app/src/components/PostContainer/PostPage.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+//import withAuthenticate from './components/Authentication/withAuthenticate';
+import '../../App.css';
+import SearchBar from '../SearchBar/SearchBar';
+import dummyData from '../../dummy-data';
+import PostContainer from './PostContainer';
+
+class PostPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      comments: {
+        text: '',
+        username: ''
+      }
```

Comment:
It seems that this `comments` object in `this.state` is used for holding information on a new comment. Therefore, it might be better to rename the `comments` object into something more declarative (like `newComment`, to provide one idea).